### ISSUE TITLE
Default selected time is clipped (24H Format)

### ISSUE THREAD

**Issue by GeorgeXCV** [itsmeichigo/DateTimePicker#95]
**Describe the bug**
When I open the picker, the default time selected (current time) is clipped underneath the date selection. I am using the default width / height.

```
 picker = DateTimePicker.create(minimumDate: min, maximumDate: max)
            picker.frame = CGRect(x: 0, y: 250, width: picker.frame.size.width, height: picker.frame.size.height)
            picker.is12HourFormat = false
            picker.dateFormat = "dd/MM/YYYY HH:mm"
```

**To Reproduce**
1. Add DatePicker to view using 24H Format.

**Expected behavior**
See 2nd screenshot.

**Screenshots**
1st screenshot is what I see everytime. 2nd screenshot is what I expect.
<img width="327" alt="Screenshot 2020-01-06 at 19 47 26" src="https://user-images.githubusercontent.com/13811547/71844508-350a2c00-30be-11ea-999e-30030cb29f1f.png">
<img width="328" alt="Screenshot 2020-01-06 at 19 47 43" src="https://user-images.githubusercontent.com/13811547/71844516-389db300-30be-11ea-966f-5d2fec33879c.png">

**Library and Device details (please complete the following information):**
 I tried multiple device simulators, iPhone SE and iPhone 8. iOS13.

**Comment by itsmeichigo** [comment]
Hi @GeorgeXCV this has been fixed, please try again with version 2.4.0 and report back if the issue persists.

**Comment by GeorgeXCV** [comment]
Unable to verify since updating has broke all functionality:
<img width="1275" alt="Screenshot 2020-03-01 at 16 39 51" src="https://user-images.githubusercontent.com/13811547/75629607-50377a80-5bdb-11ea-9cc8-3ab4b04871fc.png">

**Comment by itsmeichigo** [comment]
Can you share more detailed crashlog? I can't seem to reproduce the issue.

**Comment by Nwachuks** [comment]
Hi. Please I'm using version 2.4.1 and it's giving me this.
<img width="954" alt="Screenshot 2020-08-19 at 2 16 33 AM" src="https://user-images.githubusercontent.com/41437101/90581081-483bb900-e1c2-11ea-9e08-37e7aec1cb06.png">

**Comment by itsmeichigo** [comment]
Closing this issue due to inactivity. If you're using SPM, the crash has been fixed in v2.5.0.
